src/lib/bootloader/stage1.rb

```diff
@@ -158,20 +160,20 @@ def available_locations
     end
 
     def can_use_boot?
-      tm = Yast::Storage.GetTargetMap
       partition = Yast::BootStorage.BootPartitionDevice
 
-      part = Yast::Storage.GetPartition(tm, partition)
+      parts = partitions.with(name: partition)
 
-      if !part
+      if parts.empty?
         log.error "cannot find partition #{partition}"
         return false
       end
 
-      log.info "Boot partition info #{part.inspect}"
+      log.info "Boot partition info #{parts.first.inspect}"
 
       # cannot install stage one to xfs as it doesn't have reserved space (bnc#884255)
-      return false if part["used_fs"] == :xfs
+      fs = parts.filesystems.first
```

Comment:
ok, so result of not so nice api of this filesystem call. OK. So just result of my below note.